dart/test/sentry_client_test.dart

```diff
@@ -1033,6 +1033,128 @@ void main() {
     });
   });
 
+  group('SentryClient ignored errors', () {
+    late Fixture fixture;
+
+    setUp(() {
+      fixture = Fixture();
+      fixture.options.ignoreErrors = ["my-error", "error-.*"];
+    });
+
+    test('drop event if error message fully matches ignoreErrors value',
+        () async {
+      final event = SentryEvent(message: SentryMessage("my-error"));
+
+      final client = fixture.getSut();
+      await client.captureEvent(event);
+
+      expect((fixture.transport).called(0), true);
+    });
+
+    test('drop event if error message partially matches ignoreErrors value',
+        () async {
+      final event = SentryEvent(message: SentryMessage("error-foo"));
+
+      final client = fixture.getSut();
+      await client.captureEvent(event);
+
+      expect((fixture.transport).called(0), true);
+    });
+
+    test('send event if error message does not match ignoreErrors value',
+        () async {
+      final event = SentryEvent(message: SentryMessage("warning"));
+
+      final client = fixture.getSut();
+      await client.captureEvent(event);
+
+      expect((fixture.transport).called(1), true);
+    });
+
+    test(
+        'send event if error message does only partially match ignoreErrors value',
+        () async {
+      final event =
+          SentryEvent(message: SentryMessage("partially matching my-error"));
```

Comment:
maybe it's just my misunderstanding, shouldn't this case be dropped?

> When using strings, partial matches will be filtered out

I might be wrong though

Reply:
@martinhaintz